### ISSUE TITLE
Tensorflow has not supported operator [Elu] with name [elu1/Elu].

### ISSUE THREAD

**Issue by Zrufy** [microsoft/MMdnn#873]
Platform (like ubuntu 16.04/win10):
Windows 10

Python version:
3.6

Source framework with version (like Tensorflow 1.4.1 with GPU):Tensorflow 1,14

Destination framework with version (like CNTK 2.3 with GPU):Onnx

Pre-trained model path (webpath or webdisk path):

Running scripts:
mmtoir -f tensorflow -w model.pb --inNodeName input_1 --inputShape 224,224,3 --dstNodeName act_softmax/Softmax -o mobilenet

**Comment by linmajia** [comment]
@Zrufy , thank you very much for the feedback. Are you using TF mobilenet? It seems to use Relu6 only instead of Elu.

**Comment by Zrufy** [comment]
I was trying to bring the ssd model of this project to pytorch or onnx. I encountered many difficulties. However I solved it by transforming the activations from elu to relu.

**Comment by JiahaoYao** [comment]
Hi @Zrufy, would you like to share your original model with ELU?
